### PR TITLE
Zowe PSWI docs

### PR DESCRIPTION
**Pull request by zeibura**
### Description (including links to related git issues)
Please describe your pull request.
https://github.com/zowe/docs-site/issues/1912

This is the doc update for the z/OS installation with PSWI. I have added 4 new pages and updated the main installation roadmap.

The following still needs to be done after this PR is merged:
- Updating the diagram in the installation roadmap (Igor Kazmyr has the new diagram in the ppt and will generate it)
- The 4 new pages need to be added to the sidebar. I don't know where that comes from. The page "install-zowe-pswi.md" will be a subpage of "installing" and the other three will be subpages of install-zowe-pswi in the following order:
    - install-zowe-pswi.md
        - install-zowe-pswi-address-requirements.md
        - install-zowe-pswi-acquire.md
        - install-zowe-pswi-deployment.md

### CODE REVIEW

File: docs/user-guide/install-zowe-pswi-acquire.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+## Download the Portable Software Instance from Zowe Downloads
+
+To acquire the portable software instance, you download it from the Zowe Downloads page and transfer it to a local z/OSMF host using a file transfer utility, such as FTP.
+
+The portable software instance is a portable form of a software instance, including the SMP/E CSI data sets, all associated SMP/E-managed target and distribution libraries, non-SMP/E-managed data sets, and meta-data that is required to describe the product software instance.
+
+1. Go to [Zowe Downloads](https://www.zowe.org/download.html) and find **Zowe - Portable Software Instance**.
+2. Download the latest version of the package to your workstation.
+3. Use an file transfer utility such as an FTP client to transfer the single pax file to the mainframe.
+4. Execute JCL to unpack the installation file and restore the individual pax files. Sample JCL follows:
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi-acquire.md:25]: 4. Execute the JCL

File: docs/user-guide/install-zowe-pswi-address-requirements.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      * Confirm that you have write authority to the USS directories that are used for the z/OSMF pax installation process.
+      * Confirm that you have available USS file space.
+   * To download and unpack the pax file, you need free space that is approximately 3.5 times the pax file size in the file system that contains the pax directories. For example, to download and unpack a 14-MB pax file, you need approximately 49 MB of free space in the file system hosting your pax directory. If you do not have sufficient free space, error messages like EZA1490I Error writing to data set or EZA2606W File I/O error 133 can occur.   
+
+* **Configure SMP/E Internet Service Retrieval**
+    * **Role**: Systems programmer, security administrator
+    * Configure SMP/E Internet Service Retrieval to receive and download maintenance on a regular cadence or build custom maintenance packages (order PTFs, APARs, critical, recommended, all, or just HOLDDATA). This step is our recommended best practice when installing maintenance and is required to use z/OSMF Software Update. For configuration details, see the Mainframe Common Maintenance Procedures documentation.
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi-address-requirements.md:31]: ... is required to use the z/OSMF Software Update

File: docs/user-guide/install-zowe-pswi-deployment.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+To install Zowe PSWI using z/OSMF and make the product software available for use on a system by users and other programs, define a new deployment. This step defines the SMP/E environment name and the prefix of the CSI data set in z/OSMF. You also specify data set allocation parameters for all SMP/E data sets, target libraries, and distribution libraries. To define a new deployment, complete the deployment checklist (specify the USS path, DSN, VOLSERs), and submit the deployment jobs through the z/OSMF user interface. When the deployment is complete, you have a source and target copy of the software.
+
+For more information about these tasks, see [Deploying software](https://www.ibm.com/docs/en/zos/2.4.0?topic=task-deploying-software) in the IBM documentation.
+
+Subsequent maintenance activities for the product update the SMP/E environment without affecting your active product runtime environments. You decide when to redeploy the maintenance-updated SMP/E target data sets to each of the product runtime environments.
+
+Before you begin this process, review the installation best practices and verify that you meet the installation requirements in Prepare for Installation.
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi-deployment.md:11]: I think this Prepare for Installation either needs a link or more specification as to which article you are referencing.

File: docs/user-guide/install-zowe-pswi-deployment.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+1. Display the Deployments table in z/OSMF (**Software ManagementU**, **Deployments**).
+2. Define a new deployment by selecting **New** from the Actions menu.  
+The deployment checklist displays. You can also modify, view, copy, cancel, or remove existing deployments.
+3. Complete the deployment checklist items as described in Defining new deployments in the IBM documentation.  
+As you complete the deployment checklist, be sure to make the following selections:
+    1. Specify the properties for this deployment (name, description, and optional category).
+    2. Select the software to deploy.  
+    For this step, select the **Portable Software Instance** button and select your package.
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi-deployment.md:22]: Maybe remove "button"

File: docs/user-guide/install-zowe-pswi-deployment.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        - For **Volumes and Storage Classes**, no action is required. A summary is presented of the target data sets to be created and how much space is required.
+        - For **Mount Points**, review the mount points for the UNIX file system data sets that are included in the target software instance. When specifying a new target mount point, retain the static path extension in the path name to prevent failures in the configuration workflow. For example, _targetpathname_**/staticpathextension**.
+        - **Note**: If your product does not include USS directories, ignore this instruction.
+    6. Define the job settings to generate JCL to install the software and view the deployment summary. For this step, update the JOB statement as needed.
+        - **Note**: If the target system for the deployment is in a JES Multi-Access Spool (MAS) and the mount point is only accessible from the target system, add a System Affinity (SYSAFF) to the job card to ensure execution on the system where the zFS resides.
+    7. Submit the deployment jobs in sequential order, wait for each job to complete, and then select Refresh to register job completion in z/OSMF.
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi-deployment.md:40]: Refresh should be bold

File: docs/user-guide/install-zowe-pswi-deployment.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        - **Note**: If your product does not include USS directories, ignore this instruction.
+    6. Define the job settings to generate JCL to install the software and view the deployment summary. For this step, update the JOB statement as needed.
+        - **Note**: If the target system for the deployment is in a JES Multi-Access Spool (MAS) and the mount point is only accessible from the target system, add a System Affinity (SYSAFF) to the job card to ensure execution on the system where the zFS resides.
+    7. Submit the deployment jobs in sequential order, wait for each job to complete, and then select Refresh to register job completion in z/OSMF.  
+    A return code of zero is expected.  
+    When all deployment jobs have executed successfully, you have unzipped and renamed the product data sets, and updated the CSI data sets. The product data sets have also been successfully copied and the properties for the target software instance have been specified.
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi-deployment.md:42]: Consider changing verb tense to present.

File: docs/user-guide/install-zowe-pswi.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+# Installing Zowe from a Portable Software Instance
+
+As a systems programmer, your responsibilities include acquiring, installing, maintaining, and configuring mainframe products on your systems. z/OSMF lets you perform these tasks. z/OSMF lets you manage software on your z/OS systems through a browser at any time, from any location. By streamlining some traditional tasks and automating others, z/OSMF can simplify some areas of system management and also reduce the level of expertise that is required for managing system activities. Experienced users can view, define, and update policies that affect system behavior, monitor system performance, and manage their z/OS software.
+As products and vendors adopt z/OSMF services, you can install and maintain all your mainframe products in a common way according to industry best practices. After configuration is complete, you can execute the product and easily provision new software instances for use on other systems throughout your environment.
+
+## Prerequisites
+
+To install Zowe using z/OSMF, ensure that you have:
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi.md:8]: To install Zowe using z/OSMF, ensure that you meet the following requirements:

File: docs/user-guide/install-zowe-pswi.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+Provides information about z/OSMF general configuration and security requirements.
+* **Acquire a z/OSMF Portable Software Instance**  
+Provides the steps to acquire the product software by downloading the z/OSMF portable software instance to the z/OSMF host. You must then register the portable software instance in z/OSMF.
+* **Install Product Software Using z/OSMF Deployments**  
+Provides the steps to install (deploy) the portable software instance to an LPAR using z/OSMF Deployments. This step creates the SMP/E environment and runs the RECEIVE, APPLY, and ACCEPT steps to prepare the software instance for SMP/E operations. This step also:
+    * Customizes the data set names that are defined to SMP/E.
+    * Mounts required USS files if necessary.
+    * Performs workflow execution to customize the deployed runtime environment for use on a specific z/OS system.
+* **Import Product Information into z/OSMF**  
+Provides the steps to import the product information file so that z/OSMF has access to the latest product information. For a list of products using z/OSMF, see Mainframe Products using z/OSMF in the Mainframe Common Maintenance Procedures documentation.
```
Comment (janan07) [inline docs/user-guide/install-zowe-pswi.md:28]: Consider adding a link to Mainframe Products using z/OSMF in the Mainframe Common Maintenance Procedures

https://techdocs.broadcom.com/us/en/ca-mainframe-software/traditional-management/mainframe-common-maintenance-procedures/1-0/getting-started/z-osmf-requirements/mainframe-products-using-z-osmf-for-software-management.html

**Review by janan07 — APPROVED** [review]

File: docs/user-guide/install-zowe-pswi.md (hunk 8 above)
Comment (zeibura) [reply docs/user-guide/install-zowe-pswi.md:28]: I am just going to remove this, it's from the template but I don't think it's necessary here

File: docs/user-guide/install-zowe-pswi-deployment.md (hunk 3 above)
Comment (zeibura) [reply docs/user-guide/install-zowe-pswi-deployment.md:11]: removed - these refer to absent pages in the template

**Comment by nannanli** [comment]
@janan07 @zeibura The new topics were not added to the sidebar/navigation. Would you please update this file to include them so this PR could be merged? https://github.com/zeibura/docs-site/blob/PSWI-docs/sidebars.js

**Comment by zeibura** [comment]
> @janan07 @zeibura The new topics were not added to the sidebar/navigation. Would you please update this file to include them so this PR could be merged? https://github.com/zeibura/docs-site/blob/PSWI-docs/sidebars.js

Done, please check I've done this correctly

**Review by janan07 — APPROVED** [review]
Looks good

**Pull request merged**